bower.json

```diff
@@ -29,6 +29,7 @@
     "core-doc-viewer": "Polymer/core-doc-viewer#0.8-preview",
     "core-icon": "Polymer/core-icon#0.8-preview",
     "core-icons": "Polymer/core-icons#0.8-preview",
+    "test-fixture": "Polymer/test-fixture#master",
```

Comment:
I think that's a good practice. It is made painful by the way that we treat elements (as separate projects in different repos). However, the elements are an explicitly interdependent body of code. If Polymer has a new version, and we want to update an element to that version, then all of that element's dependencies should also be updates where necessary.

And an update: I have tagged 0.8.0 in polymerelements/test-fixture. You should be able to depend on that reliably.